### PR TITLE
added initial files for ui-kit

### PR DESCRIPTION
**Pull request by iwwwi**
# What does this PR do:

Adds initial project files: 

- package.json
- webpack.config.js
- components (.js, .scss and .md)
- tests
- style guide web service
- scripts
- licence

# Where should the reviewer start: 

- README.md
- package.json

### CODE REVIEW

**Comment by fahad19** [comment]
please consider having an `.editorconfig` file too: https://github.com/Travix-International/frint/blob/master/.editorconfig

that would take care of having line endings for example:

screen shot 2017-01-16 at 14 23 29

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+- `npm run styleguide-build` to build web service with living style guide
+- `npm run styleguide-server` to run web service with living style guide
+- open [localhost:3000](http://localhost:3000/)
+
+### Development
+
+#### Start developing
+
+- `npm run build:watch` to build the themes, styles and javascript on each file change
+- `npm run build:theme` to build the themes on each theme change
```
Comment (asci) [inline README.md]: also watch available

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
+- `npm run cov` to run unit test with coverage report
+- `npm run update-snapshots` to update current unit test snapshots
+- `npm run lint` to check ES-lint errors
+
+#### Requirements
+
+
+- Developer must follow the sturcture of the project 
+  - every component must contain own directory with its own `.js`, `.scss` and `.md` files
+- 100% unit test coverage of components and helpers
```
Comment (asci) [inline README.md]: to ask developers do it we need to have it from start, please adjust tests

File: getThemeYaml.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
```
Comment (asci) [inline getThemeYaml.js]: please add comment what is this file about

Comment (iwwwi) [inline getThemeYaml.js]: Better to remove this file :) this was example of usage on `travix-themes`. Which is unrelated to ui-kit itself.

File: README.md (hunk 2 above)
Comment (iwwwi) [inline README.md]: fixed

**Pull request merged**